### PR TITLE
Add Upload File Box for testing upload scenario

### PR DESCRIPTION
**Pull request by anmaur**
### Main changes in the PR:

1. Added upload File boxAndButton for testing default file upload scenario in webviews/iframe's using `<input type="file">`
2. Added checkbox to enable uploading multiple files
3. Added this JSX element inside FilesAPIs.tsx to reflect a file related scenario.

### Validation performed:

1. Load test app and click on "Choose File" under `File Upload` box
2. It should upon up the native file chooser popup.
4. Select any file and choose open to get the file-name reflected in the output section of `File Upload` box

### Unit Tests added:

> Unit tests are required for all changes. If no unit tests were added as part of this change, please explain why they aren't necessary.

<Yes/No> -> No, Its a test app change

### End-to-end tests added:

<Yes/No> -> No,  Its a test app change

### Change file added:

> Ensure the change file meets the [formatting requirements](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball).

<Yes/No>, -> No, Its a test app changes

### CODE REVIEW

**Review by jadahiya-MSFT — APPROVED** [review]

File: apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx [hunk 1]
```diff
@@ -1,9 +1,46 @@
...
 import { ApiWithCheckboxInput, ApiWithoutInput, ApiWithTextInput } from '../utils';
+import { ApiContainer } from '../utils/ApiContainer';
 import { ModuleWrapper } from '../utils/ModuleWrapper';
 
+export const FileUpload: React.FC = () => {
+  const fileUploadName = 'fileUpload';
+  const [result, setResult] = React.useState<string>('');
+  const onChangeCallback = React.useCallback(
+    (event: ChangeEvent<HTMLInputElement>) => {
+      setResult(noHostSdkMsg);
```
Comment (maglims) [inline apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx:14]: with this, will it show that "no sdk response" message first then flip one of the results? (line 22, or 25_

**Review by maglims — APPROVED** [review]

File: apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx (hunk 1 above)
Comment (anmaur) [reply apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx:14]: yes, that is correct.

Comment (maglims) [reply apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx:14]: just for my curiosity, is it any specific reason we need this default?

Comment (anmaur) [reply apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx:15]: It essentially states out that API was called from Teams test app successfully but did not get any response. There are cases when API call is halted at test app level eg. when the param is not valid

File: apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx [hunk 2]
```diff
@@ -1,9 +1,60 @@
 import { FileOpenPreference, files, SdkError } from '@microsoft/teams-js';
-import React, { ReactElement } from 'react';
+import React, { ChangeEvent, ReactElement } from 'react';
 
+import { noHostSdkMsg } from '../../App';
 import { ApiWithCheckboxInput, ApiWithoutInput, ApiWithTextInput } from '../utils';
+import { ApiContainer } from '../utils/ApiContainer';
 import { ModuleWrapper } from '../utils/ModuleWrapper';
 
+export const FileUpload: React.FC = () => {
```
Comment (jekloudaMSFT) [inline apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx:9]: Does this control call teams-js? It looks like it's just getting files via the HTML input and then displaying the result, is that right?

Comment (anmaur) [reply apps/teams-test-app/src/components/privateApis/FilesAPIs.tsx:9]: It doesnot. Essentially on any iframe/webview for <input type="file' /> you could choose and select files. This control enables testing of that

**Review by jekloudaMSFT — APPROVED** [review]

**Review by jekloudaMSFT — APPROVED** [review]

**Review by maglims — APPROVED** [review]

**Pull request merged**